src/components/Navbar.js

```diff
@@ -0,0 +1,35 @@
+import {Link} from 'react-router-dom'
+
+const Navbar = () => {
+    return (
+      <nav className='navbar navbar-expand-lg navbar-dark bg-primary'>
+        <a className='navbar-brand' href='#'>
+          Tour of Heroes
```

Comment:
Change line 7 from Tour of Heroes to Team Alkali (or something along those lines)

Reply:
revised